src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/stft.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2024 Intel Corporation
```

Comment:
Please move this file to https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/common

Reply:
Last time I've received a comment from @dmitry-gorokhov to move the cpu tests from custom/single_layer to shared ;) 
https://github.com/openvinotoolkit/openvino/pull/26007#discussion_r1726731739

> Basic recommendation is to have new tests implemented as shared ones to enable their reuse between different plugins.

Is there any difference for this case?

Reply:
Leaving as shared tests, can be moved when needed.

Reply:
Test instantiation files for `rdft` and `dft` are also in the "shared_tests_instances" directory, so it's reasonable to keep the file for `stft` in the same place. Is there any specific reason to move the `stft` file to other directory?
[/src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/rdft.cpp](https://github.com/openvinotoolkit/openvino/blob/46a761d3802c94252447d46c71ab90b1c1ebe794/src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/rdft.cpp)
[/src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/dft.cpp](https://github.com/openvinotoolkit/openvino/blob/46a761d3802c94252447d46c71ab90b1c1ebe794/src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/dft.cpp)